fsystem/src/FileSystem.cpp

```diff
@@ -273,6 +266,15 @@ bool FileSystem::endsWith(const std::string& file, const std::string& ending)
     }
 }
 
+bool FileSystem::endsWith(const std::string& file, const char ending)
+{
+    if (!file.empty() && (file[file.length() - 1] == ending)) {
```

Comment:
Can be simplified to `return !file.empty() && file[file.length() - 1] == ending;`

Reply:
yup